src/sort.js

```diff
@@ -0,0 +1,13 @@
+import CONST from './constants';
+
+
+function hotScore(votes, createdAt) {
+  let order = Math.log10(Math.max(Math.abs(votes), 1));
+  let sign = Math.sign(votes);
+  let age = createdAt - CONST.FEED_ZERO_TIME;
+  return Math.round(order + sign * age / CONST.FEED_INFLATION_INTERVAL);
+}
```

Comment:
Is there a wikipedia or other web page that explains this algorithm? Would be nice to add it as a comment. Or a short description.